lib/iiif_image_plug/v3.ex

```diff
@@ -393,6 +393,9 @@ defmodule IIIFImagePlug.V3 do
   defp send_buffered(conn, %Image{} = image, format) do
     {:ok, buffer} = Image.write_to_buffer(image, ".#{format}")
 
+    # Use the requested format for content-type (since that's what we're delivering)
+    conn = put_resp_content_type_from_format(conn, format)
+
```

Comment:
Maybe we could add these calls between line 329 and 330, before it branches off to the different response types?